_includes/tutorials/aggregating-count/kstreams/markup/dev/make-topology.adoc

```diff
@@ -0,0 +1,17 @@
+Then create the following file at `src/main/java/io/confluent/developer/AggregatingCount.java`. Let's take a close look at the `buildTopology()` method, which uses the Kafka Streams DSL.
+
+The first thing the method does is create an instance of `https://kafka.apache.org/{{ site.ak_javadoc_version }}/javadoc/org/apache/kafka/streams/StreamsBuilder.html[StreamsBuilder]`, which is the helper object that lets us build our topology. With our builder in hand, we can apply the following methods:
+
+1. Call the `stream()` method to create a `https://kafka.apache.org/{{ site.ak_javadoc_version }}/javadoc/org/apache/kafka/streams/kstream/KStream.html[KStream]<String, TicketSale>` object.
+
+2. Since we can't make any assumptions about the key of this stream, we have to repartition it explicitly. We use the `map()` method for that, creating a new `https://kafka.apache.org/{{ site.ak_javadoc_version }}/javadoc/org/apache/kafka/streams/KeyValue.html[KeyValue]` instance for each record, using the movie title as the new key.
+
+3. Group the events by that new key by calling the `groupByKey()` method. This returns a `https://kafka.apache.org/{{ site.ak_javadoc_version }}/javadoc/org/apache/kafka/streams/kstream/KGroupedStream.html[KGroupedStream]` object.
+
+4. Use the `reduce()` method to apply the count aggregation.
```

Comment:
The code in `AggregatingCount.java` uses `count()` rather than `reduce()`, which would need to be passed a reducer to do the job.

Reply:
@colinhicks great catch.  I have modified the adoc to refer to `count()` (rather than `reduce()`). For clarification, are you suggesting there is another change required to the code, or the code is fine?

Reply:
@ybyzek, cool! The code is fine as is. We can mark this resolved.